src/context/types.ts

```diff
@@ -0,0 +1,11 @@
+export interface IAuthContextType {
```

Comment:
Naming is misleading. IAuthContextType says it's a type, but actually it's an interface.